contributing/diversity/index.rst

```diff
@@ -0,0 +1,7 @@
+Community
```

Comment:
Maybe change the title of this page to *"Symfony Diversity"* or *"Symfony Diversity & Inclusion"* or *"Symfony Diversity & Inclusion Initiative"*?

Reply:
Sorry, was using `/contributing/community/index.rst` as a template for the formatting and unconsciously retyped *Community*!

Reply:
I've named it *Diversity Initiative* (rather than *Symfony Diversity Initiative*) to follow the convention of similar titles *Code of Conduct* and *Community*.